generate_configurations.py

```diff
@@ -93,7 +93,7 @@ def requirements(env):
     yield 'pytest==%s' % (env.pytest_version)
     yield 'pytest-xdist==1.13.1'
     yield DJANGO_REQUIREMENTS[env.django_version]
-    yield 'django-configurations==0.8'
+    yield 'git+https://github.com/jezdez/django-configurations'
```

Comment:
Thanks! :)
(but is redirected currently and will be replaced by 1.0 soonish.. ;)